alerts/src/iosMain/kotlin/Alerts.kt

```diff
@@ -32,14 +34,31 @@ actual class AlertInterface(
     private val parent: UIViewController
 ): BaseAlertPresenter(alert) {
 
-    override fun show(animated: Boolean, completion: (() -> Unit)?) {
+    override fun show(animated: Boolean, completion: (() -> Unit)) {
+        showAlert(completion = completion)
+    }
+
+    override suspend fun show(): Alert.Action? = suspendCancellableCoroutine { continuation ->
```

Comment:
This method is also the same as it's Android counterpart except for `dismiss()` though it seems to me the Android version can use `dismiss()` also.

Reply:
Now it's fixed, please take a look